xbe/__init__.py

```diff
@@ -673,6 +673,19 @@ def _init_from_data(self, data):
 			# Get section name
 			sec_name = self.get_cstring_from_offset(data, self.vaddr_to_file_offset(sec_hdr.section_name_addr), 'ascii')
 
+			# Check for duplicate section names and rename
+			count = 1
+			sec_name_base = sec_name
+			while sec_name in self.sections:
```

Comment:
Why two loops?

Reply:
Pushed the wrong code, sorry about that. I've pushed the correct code